src/components/RequesterPaysModal.js

```diff
@@ -12,27 +12,28 @@ import { requesterPaysProjectStore } from 'src/libs/state'
 import * as Utils from 'src/libs/utils'
 
 
-const billingHelpInfo = div({ style: { paddingTop: '1rem' } }, [
+const requesterPaysHelpInfo = div({ style: { paddingTop: '1rem' } }, [
   h(Link, {
     href: 'https://support.terra.bio/hc/en-us/articles/360029801491',
     ...Utils.newTabLinkProps
-  }, ['Why is billing required to access this data?', icon('pop-out', { style: { marginLeft: '0.25rem' }, size: 12 })])
+  }, ['Why is a workspace required to access this data?', icon('pop-out', { style: { marginLeft: '0.25rem' }, size: 12 })])
 ])
 
 const RequesterPaysModal = ({ onDismiss, onSuccess }) => {
   const [loading, setLoading] = useState(false)
-  const [billingList, setBillingList] = useState([])
-  const [selectedBilling, setSelectedBilling] = useState(requesterPaysProjectStore.get())
+  const [workspaceList, setWorkspaceList] = useState([])
+  const [selectedGoogleProject, setSelectedGoogleProject] = useState(requesterPaysProjectStore.get())
   const signal = Utils.useCancellation()
 
   Utils.useOnMount(() => {
-    const loadBillingProjects = _.flow(
+    const loadWorkspaces = _.flow(
       Utils.withBusyState(setLoading),
-      withErrorReporting('Error loading billing projects')
+      withErrorReporting('Error loading workspaces')
     )(async () => {
-      setBillingList(await Ajax(signal).Billing.listProjects())
+      const workspaces = await Ajax(signal).Workspaces.list()
+      setWorkspaceList(_.filter(workspace => workspace.accessLevel === 'OWNER' || workspace.accessLevel === 'PROJECT_OWNER', workspaces))
```

Comment:
Are the `accessLevel`s here the equivalent of roles in Sam?  If so, have we verified that this is the complete list of roles we want to filter on here?

Reply:
Writers with compute access should also be able to use the workspace IMO since they have permission to use the workspace's Google project. Owners and project owners get compute access automatically

Reply:
So, I would love to be able to do that, but unfortunately 'can-compute' doesn't actually return with the workspace list response, and we don't necessarily want to query getWorkspace N times for that information. I think it could be a future change, but I am very worried about including that here for scope creep/releasing PPW in a reasonable timeframe

Reply:
https://github.com/broadinstitute/rawls/blob/develop/core/src/main/resources/swagger/api-docs.yaml#L6010 is what this accessLevel is referring to.